.circleci/config.yml

```diff
@@ -351,6 +351,11 @@ jobs:
           command: |
             EMCC_CORES=4 python3 ~/emscripten/embuilder.py build ALL
             python3 ~/emscripten/tests/runner.py test_hello_world
+      - run:
+          name: embuilder build for lto
+          command: |
+            EMCC_CORES=4 python3 ~/emscripten/embuilder.py build libc++ --lto
```

Comment:
Why not `build ALL`

Reply:
`ALL` takes around 5 minutes, and we only need libc++ for the tests we currently have. Just trying to save time on the bots.